scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -314,7 +314,14 @@ monitor_sst_progress() {
     fi
 
     if [[ ${current_timeout} -eq  ${timeout} ]]; then
+      wsrep_log_error "******************* FATAL ERROR ********************** "
       wsrep_log_error "Killing SST ($pid) with SIGKILL after stalling for ${current_timeout} seconds"
+      wsrep_log_error "This message is related to sst-idle-timeout configuration variable."
```

Comment:
Should we also tell users that they can configure it in the [sst] section of the cnf file, if the current timeout is not sufficient for them?

Reply:
I think saying that it is related to XYZ configuration variable is enough. It is straightforward that as it is configuration variable, it can be configured...